apple-catalog-parsing/build.rs

```diff
@@ -94,4 +94,8 @@ fn main() {
     println!(
         "cargo:rustc-link-search={developer_dir_path}/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx"
     );
+
+    let lib_dir = "/usr/lib/swift";
+    println!("cargo:rustc-link-arg=-Wl,-rpath");
+    println!("cargo:rustc-link-arg=-Wl,{lib_dir}");
```

Comment:
This would no longer be necessary if we restrict to ARM, correct?

Reply:
Or wait, this is the crate's build.rs, so I don't think these directives have any effect here